lib/Util/Options.cpp

```diff
@@ -255,6 +255,12 @@ namespace SVF
         llvm::cl::desc("Dump dot graph of PAG")
     );
 
+    const llvm::cl::opt<bool> Options::PAGDotGraphShorter(
+            "dump-pag-shorter",
+            llvm::cl::init(false),
```

Comment:
It looks pretty good to me for the shorter version. Please make it as default.

Reply:
I have made the shorter version the default.